esp/GBPlay/sdkconfig

```diff
@@ -63,6 +63,7 @@ CONFIG_BOOTLOADER_WDT_TIME_MS=9000
 # CONFIG_BOOTLOADER_SKIP_VALIDATE_ALWAYS is not set
```

Comment:
Did you purposefully make changes to this file (i.e., `idf.py menuconfig`), or are you just using a newer version of IDF than I am which caused stuff to get added?

I'm on v4.4-dev-2359-g58022f859 ([this commit](https://github.com/espressif/esp-idf/commit/58022f859); I ended up not taking an official release because there was a bug I needed a fix for).

We should agree on a version to avoid incompatibilities between our environments.

Reply:
I think my version of idf.py may have done this, that sounds good, it's probably best that we prevent confusion from this